packages/components/src/tools-panel/stories/index.tsx

```diff
@@ -162,93 +189,143 @@ export const WithNonToolsPanelItems = () => {
 		</PanelWrapperView>
 	);
 };
+WithNonToolsPanelItems.args = {
+	...Default.args,
+	label: 'ToolsPanel (with non-menu items)',
+};
 
-export const WithOptionalItemsPlusIcon = ( { isShownByDefault } ) => {
```

Comment:
Since `isShownByDefault` is not a prop of `ToolsPanel`, in order to adhere to how controls are supposed to work I decided to move it inside the render function as internal state (`isFirstToolsPanelItemShownByDefault`)